pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -158,6 +158,23 @@ func (self *RefsHelper) CreateGitResetMenu(ref string) error {
 		}
 	})
 
+	// Adds the custom destination menu item
+	menuItems = append(menuItems, &types.MenuItem{
+		LabelColumns: []string{
+			fmt.Sprintf("Set custom destination"),
```

Comment:
This "Set custom destination" should be internationalized as well. And while you're at it, set it to lowercase (like all the other options in the same menu). To denote that in the translations, prefix the attribute name with `Lc`.